configuration/defaults/byron-mainnet/configuration.yaml

```diff
@@ -0,0 +1,287 @@
+##########################################################
+###############          Mainnet                 #########
+############### Cardano Byron Node Configuration #########
+##########################################################
+
+
+##### Locations #####
+
+GenesisFile: genesis.json
```

Comment:
We should really modify `cardano-testnet` to be parameterized on configuration files. Currently this is hardcoded